src/mixin-props-test.js

```diff
@@ -20,3 +20,61 @@ QUnit.test("basics", function(assert) {
 	el.age = "33";
 	assert.equal(el.age, 33, "updated age");
 });
+
+try {
```

Comment:
I think all of the tests that require class fields should go in their own file. Then the main test file can do this feature detection and create a variable like `supportsClassFields`. The wrap the `require` statement for the tests like

```
if(supportsClassFields) {
  require(...);
}
```